src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/NonAssociation.kt

```diff
@@ -95,7 +95,8 @@ data class CreateNonAssociationRequest(
       reason = reason,
       restrictionType = restrictionType,
       comment = comment,
-      updatedBy = updatedBy,
+      authorisedBy = createdBy,
```

Comment:
`authorisedBy` were null when created through this endpoint, but were typically non-null free text when migrated from NOMIS

Reply:
Would `updatedBy` would be null in this case then? Is that by design?

Reply:
`updatedBy` is never null

Reply:
Tests seem to contradict this, so maybe I'm missing something as it seems like `updatedBy` would match `authorisedBy` on creation as I would expect 🤔

Reply:
(to clarify "this" refers to my comment, not yours)

Reply:
> it seems like updatedBy would match authorisedBy on creation as I would expect
yes, that's the intention of this PR. previously, `authorisedBy` would always be null when created through the api